### DISCUSSION TITLE
I can't convert a Pull Request to a draft

### DISCUSSION THREAD

**Discussion by objecttothis** [community/community#48361] [Pull Requests]
### Select Topic Area

Bug

### Body

![image](https://user-images.githubusercontent.com/17935339/220853063-6942760e-1fa6-4fc7-9aa0-eb280679bd3f.png)

I have write access to the private repository not owned by me and it shows me the link to learn about draft PR, but no button to actually do it.  If the feature is not available to private repositories, why toy with my emotions by putting a link there to learn about them?

**Comment by BRadHoc** [comment]
Having the same issue here
